bind.go

```diff
@@ -30,6 +30,19 @@ type (
 // Bind implements the `Binder#Bind` function.
 func (b *DefaultBinder) Bind(i interface{}, c Context) (err error) {
 	req := c.Request()
+
+	paramNames := c.ParamNames()
+	paramValues := c.ParamValues()
+	paramVars := make(map[string][]string)
+	for in, name := range paramNames {
+		// Fix for an echo bug where a param name would show up which dont exist if its name contains "id" e.g. "userid"
```

Comment:
@kolaente What is it about?

Reply:
When the param name is defined as `somethingid`, echo sometimes puts this as `id,something` (one string with a comma) into `c.ParamNames()`. I've tried to track this down and fix it, but I wasn't successful which is why I put that workaround here. The bug seems to occur pretty randomly. Or is that a feature?

I know it's probably not the best place to do this here.

Reply:
Can you provide steps to reproduces?

Reply:
I've registered a route to `/thing/:somethingid` and printed out `ParamNames` in the route handler after calling the default binder (before my modifications).

I wasn't able to reproduce this, with some tests right now, if you were able to, this should probably be fixed in a seperate pr before this one. If not, I'd better remove my workaround as in that case it probably was an issue with my setup.